qutebrowser/mainwindow/tabwidget.py

```diff
@@ -81,6 +81,7 @@ def _init_config(self):
         position = config.val.tabs.position
         selection_behavior = config.val.tabs.select_on_remove
         self.setTabPosition(position)
+        self.setElideMode(config.val.tabs.title.elide)
```

Comment:
The `self.setElideMode(Qt.ElideRight)` in `__init__` seems redundant with this, right? (hah, that pun)